client/container_checkpoint_opts.go

```diff
@@ -36,12 +36,8 @@ import (
 	imagespec "github.com/opencontainers/image-spec/specs-go/v1"
 )
 
-var (
-	// ErrCheckpointRWUnsupported is returned if the container runtime does not support checkpoint
-	ErrCheckpointRWUnsupported = errors.New("rw checkpoint is only supported on v2 runtimes")
-	// ErrMediaTypeNotFound returns an error when a media type in the manifest is unknown
-	ErrMediaTypeNotFound = errors.New("media type not found")
-)
+// ErrMediaTypeNotFound returns an error when a media type in the manifest is unknown
+var ErrMediaTypeNotFound = errors.New("media type not found")
```

Comment:
Wondering if we should remove this error, and instead make `GetIndexByMediaType ` return a regular error that wraps  `errdefs.ErrNotFound` https://github.com/containerd/containerd/blob/a72adffa65716a9dbbb1340d5c14a34c203a1ad4/client/container_checkpoint_opts.go#L158


This error is only checked for internally in `WithRestoreRuntim` (which could then use `errdefs.IsNotFound()` instead https://github.com/containerd/containerd/blob/a72adffa65716a9dbbb1340d5c14a34c203a1ad4/client/container_restore_opts.go#L86-L87